src/middlewared/middlewared/plugins/failover_/datastore.py

```diff
@@ -65,6 +64,11 @@ def send_retry():
                     except Exception:
                         pass
 
+                    if raise_alert_time <= 0 and self.failure:
+                        self.middleware.call_sync('alert.oneshot_delete', 'FailoverSyncFailed', None)
```

Comment:
This will delete/remove alert every `RAISE_ALERT_SYNC_RETRY_TIME` seconds which will lead to e-mails being sent. Is this intended?

Reply:
No it's not, I was just trying to avoid opening a duplicate alert if one had already been generated

Reply:
Hmm, maybe an email alert every 20mins is actually decent.